mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -2629,8 +2638,9 @@
     <string name="likert_scale_option_4" tools:ignore="UnusedResources">Dissatisfied</string>
     <!-- Option for likert scale -->
     <string name="likert_scale_option_5" tools:ignore="UnusedResources">Very Dissatisfied</string>
-
-    <!-- Microsurvey accessibility -->
+    <!-- Option for likert scale -->
+    <string name="likert_scale_option_6" tools:ignore="UnusedResources">I don\'t use it</string>
```

Comment:
```suggestion
    <string name="likert_scale_option_6" tools:ignore="UnusedResources">I don’t use it</string>
```

@t-p-white this should use an apostrophe, not an escaped straight quote.

@gabrielluong was there any progress on setting up a linter for these errors (it was originally part of the migration)?

Reply:
Thanks @flodolo. Addressed in https://phabricator.services.mozilla.com/D213398